lib/core.js

```diff
@@ -1993,6 +1997,25 @@ module.exports = function reglCore (
     }
   }
 
+  function emitBeginTimerQuery (env, draw, args) {
+    var STATS = env.stats
+    var shared = env.shared
+
+    if (timer) {
+      var TIMER = shared.timer
+      draw(TIMER, '.beginQuery(', STATS, ');')
```

Comment:
I wonder if we should reuse the same machinery for scope and draw timers?  What happens if you execute the same draw command several times in a row?  Will it call the same timer repeatedly?